### PR TITLE
Undo revert 2615

### PR DESCRIPTION
**Pull request by lrknox**

### CODE REVIEW

File: config/cmake/scripts/HDF5config.cmake [hunk 1]
```diff
@@ -38,7 +38,7 @@ cmake_minimum_required (VERSION 3.12)
 ##############################################################################
 
 set (CTEST_SOURCE_VERSION "1.10.10")
-set (CTEST_SOURCE_VERSEXT "-2")
+set (CTEST_SOURCE_VERSEXT "-1")
```
Comment (byrnHDF) [inline config/cmake/scripts/HDF5config.cmake]: Do you want to do this change?

**Review by byrnHDF — APPROVED** [review]

File: config/cmake/scripts/HDF5config.cmake (hunk 1 above)
Comment (lrknox) [reply config/cmake/scripts/HDF5config.cmake]: Probably not.  The 1_10 branch will be going to 1.10.11-1, and the 1_10_10 branch is 1.10.10-3, but I'll probably take this change out.

Comment (lrknox) [inline config/cmake/scripts/HDF5config.cmake]:
```suggestion
set (CTEST_SOURCE_VERSEXT "-2")
```

Comment (byrnHDF) [reply config/cmake/scripts/HDF5config.cmake]: Don't forget the other files that have that version issue.

**Review by byrnHDF — APPROVED** [review]

**Pull request merged**